packages/synapse-react-client/src/synapse-queries/auth/useOIDC.ts

```diff
@@ -21,10 +21,14 @@ export function useGetOAuth2RequestDescription(
     UseQueryOptions<OIDCAuthorizationRequestDescription, SynapseClientError>
   >,
 ) {
+  const { synapseClient } = useSynapseContext()
   return useQuery({
     ...options,
     queryKey: ['useGetOAuth2RequestDescription', request],
-    queryFn: () => SynapseClient.getOAuth2RequestDescription(request),
+    queryFn: () =>
+      synapseClient.openIDConnectServicesClient.postAuthV1Oauth2Description({
+        oIDCAuthorizationRequest: request,
+      }),
```

Comment:
To fix many type errors, the easiest change to make was to use the generated API client instead of the manually-written one.